src/DynamoInstall/Product.wxs

```diff
@@ -104,7 +103,7 @@
 
     <!-- Install a shortcut in the Start menu -->
     <DirectoryRef Id="ApplicationProgramsFolder">
-      <Component Id="ApplicationShortcut" Guid="d7575401-f8ba-488f-b5e4-5ad24dbc4b10" Win64="yes">
+      <Component Id="ApplicationShortcut" Win64="yes">
```

Comment:
So you are not assigning specific guid so that every install will have different guid for shortcut and then it can be automatically be removed by uninstall.